interface/globals.php

```diff
@@ -147,6 +147,8 @@ function undoMagicQuotes($array, $topLevel=true) {
 // Static assets directory, relative to the webserver root.
 // (it is very likely that this path will be changed in the future))
 $GLOBALS['assets_static_relative'] = "$web_root/public/assets";
+//Composer vendor directory, relative to the webserver root.
+$GLOBALS['vendor_dir'] = "$web_root/vendor";
```

Comment:
this is only a relative path. will need to use an absolute path:
``````
 $GLOBALS['vendor_dir'] = "$webserver_root/vendor";